snipeit.sh

```diff
@@ -391,10 +407,8 @@ case $distro in
 		curl -sS https://getcomposer.org/installer | php
 		php composer.phar install --no-dev --prefer-source
 
-		# Change permissions on directories
-		sudo chmod -R 755 $webdir/$name/storage
-		sudo chmod -R 755 $webdir/$name/public/uploads
```

Comment:
Why did we lose these chmods? Are we doing it somewhere else, or is it to be consistent with something else? Or that the `chown -R` will cover it?